### ISSUE TITLE
mail-client/thunderbird-38.8.0: mozconfig-v6.38.eclass is no longer in portage

### ISSUE THREAD

**Issue by dirk-olmes** [hhoffstaette/portage#2]
```
 *   firefox-38.8.0.ebuild, line  36:  Called inherit 'check-reqs' 'flag-o-matic' 'toolchain-funcs' 'eutils' 'gnome2-utils' 'mozconfig-v6.38' 'multilib' 'pax-utils' 'fdo-mime' 'autotools' 'virtualx' 'mozlinguas'
 *               ebuild.sh, line 284:  Called die
 * The specific snippet of code:
 *              [[ -z ${location} ]] && die "${1}.eclass could not be found by inherit()"
 * 
```

**Comment by hhoffstaette** [comment]
Aw hell..completely missed the eclass, and apparently it was treecleaned. Suggestions how to fix this or even a PR would be welcome (probably going to need the same for FF), I don't have time to look into this right away.

**Comment by hhoffstaette** [comment]
Should be fixed with commit 9965123 - thanks for the heads.up.
